.projenrc.ts

```diff
@@ -33,9 +33,9 @@ const PROJECT_NAME = 'generative-ai-cdk-constructs';
 const CDK_VERSION: string = '2.149.0';
 
 function camelCaseIt(input: string): string {
-  // Hypens and dashes to spaces and then CamelCase...
-  return input.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, _) { if (+match === 0) return ''; return match.toUpperCase(); });
-}
+   // Hypens and dashes to spaces and then CamelCase...
+   return input.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, _) { if (+match === 0) return ''; return match.toUpperCase(); });
+ }
```

Comment:
nit: consider keeping formatting consistent when possible
```suggestion
  // Hypens and dashes to spaces and then CamelCase...
  return input.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, _) { if (+match === 0) return ''; return match.toUpperCase(); });
}
```